summoner-cli/src/Summoner/Project.hs

```diff
@@ -209,7 +210,7 @@ doGithubCommands Settings{..} = do
     "git" ["add", "."]
     "git" ["commit", "-m", "Create the project"]
     unless settingsNoUpload $ do
-        "hub" $ ["create", "-d", settingsDescription, settingsOwner <> "/" <> settingsRepo]
+        "hub" $ ["create", "-d", (T.intercalate " " . lines) settingsDescription, settingsOwner <> "/" <> settingsRepo]
```

Comment:
I think it's the same as using `unwords`
```suggestion
        "hub" $ ["create", "-d", (unwords . lines) settingsDescription, settingsOwner <> "/" <> settingsRepo]
```

Reply:
Done, thanks!